src/token-builder/types.js

```diff
@@ -33,6 +33,10 @@ export class TokenBase {
     if (missingProps.length) {
       this.errors.push(`Missing properties for ${this.type} token: "${missingProps.join(', ')}"`);
     }
+
+    if (typeof this.cacheOverride === 'string' && this.cacheOverride.length > CHAR_LIMIT) {
```

Comment:
Same thing here, should we have an explicit validation that states that the `cacheOverride` property needs to be a string? Otherwise, it seems to imply that other primitive types would be valid values.

Reply:
It would make sense to throw an error here if the type isn't a string, though we haven't been too strict about type-checking for the other properties since we're expecting users to know they should be passing in strings (which would be aided by documentation).

We think this should be fine for now, and we can update our validations in a future release.